app/javascript/components/mentoring/inbox/ConversationList.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { Pagination } from '../Pagination'
 import { Conversation } from './conversation-list/Conversation'
 import { usePaginatedRequestQuery } from '../../../hooks/request-query'
+import { Loading } from '../../common/Loading'
 
 export function ConversationList({ request, setPage }) {
   const { status, resolvedData, latestData } = usePaginatedRequestQuery(
```

Comment:
Maybe also use the `isSuccess, isError, isFetching` fields here instead of status?

Reply:
Okay. I will add that to this PR.